NCMP_for_2-1-1_and_2-2-1/WHO_code_test.R

```diff
@@ -0,0 +1,97 @@
+###############################################################################################
+#
+# The user must set the working directory to match the directory of this R script
+# setwd() # uncomment this line to set the directory of the downloded script,, + csv files, keeping the folder with the data one directory above
+#
+#
+#NB: The WHO function needs to be run on each year of data separately - so 1 csv file per year
+###############################################################################################
+
+# install the package and load it in your R session:
+#install.packages("anthro",dependencies=TRUE, type="win.binary") #only need to do this once
+library(anthro) #need to run every time you start a new session, if you want to use the package
+
+
```

Comment:
I think it would be good to have all configurations here (particularly so users don't have to scroll down to find year)
so e.g.
```R
year <- "2018-19"
input_filepath <- "H:/NCMP_Rcode/ncmp_1819_final_non_disclosive_published.csv" # use forward slashes
output_filepath <- "H:/NCMP_Rcode/SDG_output"
```R

Reply:
Yes, I agree! It did cross my mind, but for some reason I didn't implement it this time round.